Functions/xASL_io_SplitASL.m

```diff
@@ -199,15 +203,28 @@ function xASL_io_SplitASL(inPath, iM0, iDummy)
         
         %% 7. Split relevant JSON parameters/arrays
         [jsonM0, jsonASL] = xASL_io_SplitASL_SplitJSON(BackupJSONPath, iM0, ASLindices, iDummy);
+
+        %% 8. Split ASL4D_aslContext.tsv
+        if xASL_exist(ASLTSVContextPath,'file') && (~isempty(iM0) || ~isempty(iDummy))
+            ASLTSVContextPathSource = strrep(ASLTSVContextPath, 'aslcontext', 'aslcontext_Source');
```

Comment:
Can you change it to `_Source_aslcontext` - to be in sync with the BackupMATPath - see line 95.

Reply:
Sure, no problem 👍